public/main/group/settings.php

```diff
@@ -55,6 +60,17 @@
 // Group name
 $form->addElement('text', 'name', get_lang('Group name'));
 
+if (!$groupMember && !api_is_allowed_to_edit(false, true)) {
+    api_not_allowed(true);
+}
+
+// Message for group rel usergroup
+if (GroupManager::is_group_linked_to_usergroup($groupEntity)) {
+    $form->addHtml('<div class="alert alert-info">'.get_lang('Warning message to warn that the user cannot modify members of linked group').'<br>
```

Comment:
A warning message should be the message to the final user, not start with "Warning message to warn...".